client/src/stories/loginSite/1-loginInput.stories.js

```diff
@@ -1,23 +1,31 @@
 import React from "react";
 import { withKnobs, select } from "@storybook/addon-knobs";
-import LoginInput from "../../components/LoginInput";
+import Input from "../../components/Input";
 import Label from "../../components/Label";
 export default {
   title: "Login Site|Input fields",
   decorators: [withKnobs]
 };
 
-export function Input() {
+export function input() {
```

Comment:
Must be a capital letter? Or is it because of your import "input"? Maybe give the function a different name.